torch/onnx/symbolic_opset11.py

```diff
@@ -917,3 +917,9 @@ def normal(g, loc, scale, seed):
     # is a sample with mean μ and variance σ's square.
     result = mul(g, scale, g.op("RandomNormalLike", loc))
     return add(g, result, loc)
+
+
+def reciprocal(g, self):
```

Comment:
I see Reciprocal was added in ONNX before opset 11, it would make sense to change opset9's definition to standardize this operation. Currently it is 
```
def reciprocal(g, self):
    return g.op("Div", torch.ones(1), self)
```